client/package.json

```diff
@@ -35,6 +35,7 @@
     "@types/react-test-renderer": "^18.0.0",
     "@types/react-virtualized-auto-sizer": "^1.0.1",
     "@types/react-window": "^1.8.5",
+    "@types/testing-library__react": "^10.2.0",
```

Comment:
why do you need this dependency for if you don't add tests?

Reply:
It was breaking the build

Reply:
can you try and remove it? It builds on my machine without it, I wanna check what happens on github actions

Reply:
@fbricon  Done with the changes ,please check on your side.